frontend/main.js

```diff
@@ -270,7 +421,14 @@ async function startApp () {
   }
 
   // register service-worker
-  await sbp('service-workers/setup')
+  /* await Promise.race(
+    [sbp('service-workers/setup'),
+      new Promise((resolve, reject) => {
+        setTimeout(() => {
+          reject(new Error('Timed out setting up service worker'))
+        }, 15e3)
+      })]
+  ) */
```

Comment:
Why is this commented out?

Reply:
It shouldn't be, but as I mentioned I needed to do this so that I could run the tests (weird issue starting the service worker)

Reply:
Uncommented